base_layer/core/src/proof_of_work/monero_rx/merkle_tree.rs

```diff
@@ -204,33 +182,21 @@ impl Default for MerkleProof {
     fn default() -> Self {
         Self {
             branch: vec![Hash::null()],
-            depth: 0,
-            path_bitmap: 0,
         }
     }
 }
 
 /// Creates a merkle proof for the given hash within the set of hashes. This function returns None if the hash is not in
-/// hashes. This is a port of Monero's tree_branch function
+/// hashes. This is a port of Monero's tcreate_merkle_proofree_branch function
```

Comment:
typo? This isn't a port of any existing monero function anyway AFAIK. 
```suggestion
/// hashes. 
```

Reply:
I would have just thought the monero function was called `tcreate_...`